### DISCUSSION TITLE
代理设定(运行脚本的设定)

### DISCUSSION THREAD

**Discussion by qqwmly** [electerm/electerm#2911] [Q&A]
通过堡垒机挑战到真正的开发环境，
操作步骤如下
新建书签，设定堡垒机IP等设定，然后在【运行脚本】处写入真正要跳转的IP，类似下图操作，如何实现呢

![image](https://github.com/electerm/electerm/assets/68533473/1d87a54f-3be2-42d3-aad8-622db2b7699a)

或者说怎么设定堡垒机跳转到实际开发机，XSHELL会设定，可是electerm设定没有找到

**Comment by zxdong262** [comment]
还没有支持，争取在未来几个版本实现
